onnx/mapping.py

```diff
@@ -96,8 +96,8 @@ def __getitem__(self, key: Union[int, str, np.dtype]) -> Any:
         else:
             warnings.warn(
                 str(
-                    f"`mapping.{self._origin_function}` is now deprecated and will be removed in the next release or so."
-                    + "To silence this warning, please use `helper.{self._future_function}` instead."
+                    f"`mapping.{self._origin_function}` is now deprecated and will be removed in a future release."
+                    + f"To silence this warning, please use `helper.{self._future_function}` instead."
```

Comment:
`+` is not needed here. Python interpreter automatically merges strings when surrounded by `()`.

Reply:
Good catch! Updated. Thanks!